lib/utils/downloadTemplateFromRepo.js

```diff
@@ -52,7 +52,7 @@ function validateUrl({ url, hostname, service, owner, repo }) {
  * @param {String} url
  */
 function isPlainGitURL(url) {
-  return url.startsWith('https') && url.endsWith('.git');
+  return (url.startsWith('https') || url.startsWith('git@')) && url.endsWith('.git');
```

Comment:
This url is then provided to [`download`](https://www.npmjs.com/package/download) module.

Is it confirmed that it supports urls starting with `git@` ?

I've just tried it locally, and it crashed for me with `Error: Basic authentication must be done with the `auth` option` error

Reply:
@medikoo If you want to use this method, you should add your SSH key to your private `gitlab` setting.

Reply:
I wonder is it good idea then to allow such urls. It requires an extra effort from user, which may not be obvious upfront, and error message is not meaningful.

Do `https://..` urls not work with gitlab? Why do you feel it's important to support `git@..` ?

Reply:
For my part, I have many private repos only can cloned by `git@xxx` format url, and I want to using this private project for serverless template. And if somebody want to use `git@xxx` url, they should know that they have ssh access right.

Reply:
@yugasun did you confirmed that this change works for you locally?

On my side it doesn't work (it crashes with `Basic authentication must be done...`), even though I can without issues clone repos via such url

Reply:
@medikoo Of course, this is a demo output:

```bash
# yugasun @ YUGASUN-MB0 in ~/Desktop/Playground [14:54:57] 
$ serverless create --template-url=git@github.com:yugasun/fullstack-application-vue-demo.git
Serverless: Generating boilerplate...
Serverless: Downloading and installing "fullstack-application-vue-demo"...
Serverless: Successfully installed "fullstack-application-vue-demo" 
```

Before, you use this format url, you should ensure that you can `git clone` it.

Reply:
Indeed I confirm it also works on my side